docker/prod/indexer/indexer.service

```diff
@@ -15,48 +15,38 @@ SEARCH_SERVER_CHILL_TIME_BETWEEN_RUNS = 900
 
 RECORDING_INDEX_MAX_AGE = 60 * 60 * 24
 
+
+def lastest_index_dir(indexes_dir, version, index_type):
+    topdir = os.path.join(indexes_dir, version)
+    dirs = [f for f in os.listdir(topdir) if os.path.isdir(os.path.join(topdir, f))]
+    dirs.sort(reverse=True)
```

Comment:
Ahha, right! Maybe just leave a comment to that effect, then feel free to merge/deploy.

Thanks!